source/nodejs/adaptivecards-designer/src/adaptive-card-schema.ts

```diff
@@ -147,6 +147,7 @@ export const adaptiveCardSchema =
           "$ref": "#/definitions/Actions"
         },
         "body": {
+          "type": ["object", "array"],
           "description": "The Card Elements to show in the primary card region",
```

Comment:
Add a Carousel, then click x to delete it, we may need to convert the content back to normal AC card instead of an weird state

![image](https://user-images.githubusercontent.com/6290692/179061702-c3ef807e-8b28-4f08-86d8-80df34617f77.png)

Reply:
not sure what "normal AC" means. I believe that if you delete the carousel - the *whole* carousel (including any pages) should be deleted. IF it makes sense - it might be good to give some warning as the user may have intended to delete just the page and not the whole carousel.

Reply:
I agree with @KesemZ here on the scope of deletion behavior. It's worth noting that the `Undo` button in this situation works perfectly. This behavior is also in line with what happens if you delete a toplevel container -- all of its children will also be removed.

My only slight nitpick here is that it's hard (impossible?) to select a non-empty `CarouselPage` on the designer surface using the mouse. Works fine with keyboard or using mouse in concert with the Card Structure panel. It's pretty corner case-y so I don't think it's higher than a P3 :)

Reply:
@paulcam206 We can fix this by adding padding to the carousel pages. Not sure if we want to enforce that styling though, so I didn't include any changes

I can also file an issue for this one since its P3 :)

Reply:
yeah, I'd just file a bug and move on :)

Reply:
Filed a bug for this issue https://github.com/microsoft/AdaptiveCards/issues/7737